### PR TITLE
Grades ui

### PR DESCRIPTION
**Pull request by ArjunK01**
# What I did
Just showing progress on grades ui 
Added chart.js and did some bootstrap formatting

## Issues addressed

# Screenshots
Screen Shot 2020-10-18 at 12 09 30 AM

### CODE REVIEW

**Review by j-alicia-long — COMMENTED** [review]
@ArjunK01 @alex-shen1 Y'all consulted and decided to import the Chart.js bundle in order to avoid using npm, right? 

We should consult someone to see if it's better to use npm or do this...Jihoon's PR (#153 ) is currently blocked for the same reason.
@brian-yu @bradknaysi @saikonuri ?

File: .env 2.travis [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Dev settings
```
Comment (Bartleby2718) [inline .env 2.travis:1]: Is this file supposed to be committed?

File: LICENSE 2 [hunk 2]
```diff
@@ -0,0 +1,674 @@
+                    GNU GENERAL PUBLIC LICENSE
```
Comment (Bartleby2718) [inline LICENSE 2:1]: Is this file supposed to be committed?

File: pull_request_template 2.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+# What I did
```
Comment (Bartleby2718) [inline pull_request_template 2.md:1]: Is this file supposed to be committed?

File: tcf_website/static/about/history/history 2.css [hunk 4]
```diff
@@ -0,0 +1,115 @@
+/*
```
Comment (Bartleby2718) [inline tcf_website/static/about/history/history 2.css:1]: Also looks like a duplicate

**Review by Bartleby2718 — CHANGES_REQUESTED** [review]
There are so many duplicate files that it is hard to identify which changes are actual changes. Please let me know when you're done deleting them.

**Pull request closed without merging**